recipes/stack-base/ubuntu/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt-get update && \
     mkdir /var/run/sshd && \
     sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd && \
     echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
-    useradd -u 1000 -G users,sudo -d /home/user --shell /bin/bash -m user && \
+    useradd -u 1000 -G users,sudo -g 0 -d /home/user --shell /bin/bash -m user && \
```

Comment:
If this modification (`-g 0`) is necessary it should be necessary for debian and centos too.

Reply:
Ah, that was left after some experiments from yesterday. Removed it